pastel/dupe_detection.go

```diff
@@ -32,6 +32,10 @@ type DDAndFingerprints struct {
 	Maxes                      *DDMaxes               `json:"maxes"`
 	Percentile                 *DDPercentiles         `json:"percentile"`
 	Score                      *DDScores              `json:"score"`
+
+	// DD-Server does not directly return these comporessed fingerprints
+	// We generate and assign to this field to avoid repeated operations
+	ZstdCompressedFingerprint []byte `json:"zstd_compressed_fingerprint"`
```

Comment:
should have `omitempty` in define it to prevent problem when un-marshall from json string

Reply:
added in next pr